docs/docs/caching.md

```diff
@@ -8,23 +8,29 @@ Different types of resources are cached differently. Let's examine how the diffe
 
 ## HTML
 
-HTML files should never be cached. When you rebuild your Gatsby site, you often update the contents of HTML files. Because of this, browsers should be instructed to check on every request if they need to download a newer version of the HTML file.
+HTML files should never be cached by the browser. When you rebuild your Gatsby site, you often update the contents of HTML files. Because of this, browsers should be instructed to check on every request if they need to download a newer version of the HTML file.
 
-The `cache-control` header should be `cache-control: public, max-age=0, must-revalidate`
+The `cache-control` header should be `cache-control: public, max-age=0, must-revalidate`<sup>1</sup>
```

Comment:
I'm not sure if Gatsby has a recommended style for something akin to footnotes. This is potentially moot based on the other comment.